tests/utils/inputparams/testparams.go

```diff
@@ -142,3 +144,8 @@ func GetTestConfig() *TestConfig {
 
 	return config
 }
+
+// GetTenantNameWithTimeStamp prepares unique tenant name by appending current time-stamp value
+func GetTenantNameWithTimeStamp(vmgroupName string) string {
```

Comment:
you know there is an open issue of avoid using `strconv.FormatInt(time.Now().Unix(), 10)` approach and use random string instead.